ams/dashboard/src/views/catalogs/Detail.vue

```diff
@@ -313,6 +313,14 @@ const authConfigMap = {
   'auth.kerberos.krb5': 'Kerberos Krb5'
 }
 
+const defaultPropertiesMap = {
+  ams: ['warehouse'],
+  hadoop: ['warehouse'],
+  custom: ['catalog-impl'],
+  glue: ['warehouse', 'lock-impl', 'lock-table'],
```

Comment:
It should be `lock.table`, refer to https://iceberg.apache.org/docs/latest/configuration/#lock-catalog-properties

Reply:
Thank you for your feedback, I will fix it ASAP.